packages/gatsby-recipes/src/providers/npm/package.js

```diff
@@ -159,9 +195,23 @@ const destroy = async ({ root }, resource) => {
     return undefined
   }
 
-  await execa(`yarn`, [`remove`, resource.name, `-W`], {
-    cwd: root,
-  })
+  if (isJest) {
+    const packageJsonPath = path.join(root, `package.json`)
+    await new Promise(resolve => {
+      mockPackageInstall.remove({
+        package: { name: resource.name },
+        nodeModulesDir: path.join(root, `node_modules`),
+        targetPackage: packageJsonPath,
+        isDev: resource.dependencyType === `development`,
+      })
+      // Make mock async
+      setTimeout(() => resolve(), Math.random() * 100)
+    })
+  } else {
+    await execa(`yarn`, [`remove`, resource.name, `-W`], {
+      cwd: root,
+    })
+  }
```

Comment:
Same applies here, mock execa. Does this mean this recipe only works in yarn land?

Reply:
`destroy` isn't supported in general for recipes yet as we need a state file which tracks what things are created in order to be sure we know how to destroy all connected resources. `destroy` is only used for tests essentially atm which is why NPM doesn't have a code path yet.